test/ServerFixture.cc

```diff
@@ -21,15 +21,78 @@
 #endif
 
 #include <boost/algorithm/string.hpp>
+#include <boost/thread/locks.hpp>
+
 #include <stdio.h>
 #include <string>
 #include <cmath>
 
+#include <ignition/math/Helpers.hh>
 #include "gazebo/gazebo.hh"
 #include "gazebo/test/ServerFixture.hh"
 
 using namespace gazebo;
 
+namespace bir {
```

Comment:
I which we had an internal math library, would make it a lot easier

Reply:
There is no need for this workaround. We can reuse the original [ServerFixture](https://github.com/osrf/gazebo/blob/gazebo7/gazebo/test/ServerFixture.hh). Check https://github.com/Brazilian-Institute-of-Robotics/models_sdf_publisher_gazebo/pull/2 and https://github.com/Brazilian-Institute-of-Robotics/forward_looking_sonar_gazebo/pull/27